### PR TITLE
API Server start speed up - take 2

### PR DESCRIPTION
**Pull request by ericsong**
There was a bug with the first revision where the generate xml step would always use the vars from db/vars.env.

We now use the vars in the environment and added steps on how to load the db/vars.env to your local setup to the README

### CODE REVIEW

File: api/libproject/devstart.rb [hunk 1]
```diff
@@ -220,6 +250,7 @@ def start_local_api()
   setup_local_environment
   common = Common.new
   common.status "Starting API server..."
+  ENV["OVERWRITE_WORKBENCH_DB_HOST"] = "true"
```
Comment (gjuggler) [inline api/libproject/devstart.rb]: What does this line do? I can't find any references to this string elsewhere.

File: api/db/load_vars.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```
Comment (gjuggler) [inline api/db/load_vars.sh]: I'd add a line or two of comments here, providing context on what this script is and when / from where it should expect to be run. Just for posterity.

File: api/libproject/devstart.rb (hunk 1 above)
Comment (ericsong) [reply api/libproject/devstart.rb]: ah, should have been removed. good catch

**Comment by ericsong** [comment]
addressed the comments here: https://github.com/all-of-us/workbench/pulls

**Pull request merged**